.storybook/main.js

```diff
@@ -0,0 +1,51 @@
+import { dirname, join } from "path";
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const config = {
+    stories: [
+        '../packages/*/stories/*.stories.js',
+        '../tools/*/stories/*.stories.js',
+    ],
```

Comment:
```suggestion
    stories: [
		{
			directory: '../packages',
			files: '*/stories/*.@(stories.js|mdx)',
			titlePrefix: 'Components',
		},
		{
			directory: "./tools",
			files: '*/stories/*.@(stories.js|mdx)',
			titlePrefix: 'Tools',
		},
    ],
```

Would it be helpful to sort the imports into the Component & Tools categories?

Reply:
Sure! I went ahead and did this.